.github/workflows/codeql-analysis.yml

```
@@ -1,67 +1,67 @@
-
-    # ✏️ If the Autobuild fails above, remove it and uncomment the following three lines
-    #    and modify them (or add more) to build your code if your project
-    #    uses a compiled language
-
-    #- run: |
-    #   make bootstrap
-    #   make release
-
-    - name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@v3
+# For most projects, this workflow file will not need changing; you simply need
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)